android/variables.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     minSdkVersion = 22
     compileSdkVersion = 33
```

Comment:
@Vlad00b does the property compileSdkVersion also need to change to 34, like WL and WW project?

Reply:
@cloud-chiu I tried change it, but I got an error while was building the app, so I decided to change only the target version and I think it will be enough for GooglePlay